opsdroid/__main__.py

```diff
@@ -160,14 +171,25 @@ def welcome_message(config):
     default=False,
     flag_value="log",
     expose_value=False,
-    help="Opens opsdroid logs with your favorite editor" " and exits.",
+    help="[Deprecated] Opens opsdroid logs with your favorite editor and exits.",
 )
-def main():
+def cli(ctx):
     """Opsdroid is a chat bot framework written in Python.
 
     It is designed to be extendable, scalable and simple.
     See https://opsdroid.github.io/ for more information.
     """
+    if ctx.invoked_subcommand is None:
+        warn_deprecated_cli_option(
+            "Running `opsdroid` without a subcommand is now deprecated. "
```

Comment:
Do you think it makes sense adding information about which commands you can run when the user types `opsdroid`? I was thinking that the deprecation message should be logged and then show all the other subcommands that can be run, that way the user could get more information on how to run other commands in case they need.

Reply:
What about something like this to let people know there is more. Rather than blurting the whole help text here?

```suggestion
            "Running `opsdroid` without a subcommand is now deprecated. "
            "Please run `opsdroid start` instead."
            "You can also run `opsdroid --help` to learn about the other subcommands"
```

Reply:
Yeah I think that looks good 👍